test/model/content_test.dart

```diff
@@ -301,6 +301,21 @@ class ContentExample {
         '\n</code></pre></div>'),
     ]);
 
+  static const lineBreaksAfterCodeBlocks = ContentExample(
+    'blank text nodes after code blocks',
+    '    code block.\n\nsome content',
+    '<div class="codehilite">'
+      '<pre><span></span><code>'
+        'code block.\n'
+      '</code></pre>'
+    '</div>\n\n'
+    '<p>'
+      'some content'
+    '</p>', [
```

Comment:
nit: let's make this test data a bit more compact: the whole `<div>` element on two lines, and the `<p>` element on one line. See the existing `codeBlock…` cases for examples.